src/lib.rs

```diff
@@ -324,18 +323,17 @@
 //!
 //! Sessions manifest to clients as cookies. These cookies have a configurable
 //! name and a value that is the session ID. In other words, cookies hold a
-//! pointer to the session in the form of an ID. This ID is a [UUID
-//! v4](https://docs.rs/uuid/latest/uuid/struct.Uuid.html#method.new_v4).
+//! pointer to the session in the form of an ID. This ID is an i128 generated by
+//! the [`rand`](https://docs.rs/rand/latest/rand) create.
 //!
 //! ### Secure nature of cookies
 //!
-//! Session IDs are considered secure if your platform's
-//! [`getrandom`](https://docs.rs/getrandom/latest/getrandom/) is
-//! secure[^getrandom], and therefore are not signed or encrypted. Note that
-//! this assumption is predicated on the secure nature of the UUID crate and its
-//! ability to generate securely-random values. It's also important to note that
-//! session cookies **must never** be sent over a public, insecure channel.
-//! Doing so is **not** secure.
+//! Session IDs are considered secure if sent over encrypted channels, and
+//! therefore are not signed or encrypted. Note that this assumption is predicated
+//! on the secure nature of the [`rand`](https://docs.rs/rand/latest/rand) crate
+//! and its ability to generate securely-random values using the ChaCha block cipher
```

Comment:
Are we still using `ChaCha` in the latest commit?

Reply:
Yes, the method use internally uses StdRng, which uses ChaCha12.